packages/browserslist-config-terra/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@cerner/browserslist-config-terra",
+  "version": "2.0.0",
```

Comment:
v3?

Reply:
We bump at release time right?

Reply:
I could probably throw some breaking change comments in there as a reminder though.

Reply:
https://github.com/cerner/terra-toolkit/pull/402/commits/688aa77b8718be36c9c8a0ac94616e88e81c12bb